src/types/Species.ts

```diff
@@ -12,8 +12,55 @@ export type Species = {
   seedStorageBehavior?: SeedStorageBehavior;
   problems?: SpeciesProblemElement[];
   ecosystemTypes?: EcosystemType[];
+  nativeStatus?: NativeStatus;
+  nativeEcosystem?: string;
+  growthFormNEXT?: GrowthForm[];
+  successionalGroup?: SuccessionalGroup[];
+  ecologicalRoleKnown?: string;
+  localUsesKnown?: string;
+  plantMaterialSourcingMethod?: PlantMaterialSourcingMethod[];
+  heightAtMaturity?: number;
+  heightAtMaturitySource?: string;
+  dbhDiameterAtMaturity?: number;
+  dbhSource?: string;
+  averageWoodDensity?: number;
+  woodDensityLevel?: WoodDensityLevel;
+  otherFacts?: string;
```

Comment:
Should a species have an array of associated ProjectIds? Or is that not part of this scope?

Reply:
I added the fields listed in the PRD here: https://terraformation.atlassian.net/wiki/spaces/PM/pages/1158021121/PRD+Species+Deliverable#Data-to-add-to-Species. `ProjectIds` doesn't appear to be one of the fields, perhaps it isn't part of this scope.